app/src/page/channel/view.js

```diff
@@ -103,7 +97,127 @@ class ChannelPage extends React.PureComponent {
                                                text={"Next"}
                                                disabled={!!fetching}
                                                onPress={this.handleNextPage} />}
-        </View>}
+        </View>
+      );
+    }
+
+    return (
+      <View style={channelPageStyle.contentTab}>
+        {contentList}
+        {pageButtons}
+      </View>
+    );
+  }
+
+  renderAbout = () => {
+    const { claim } = this.props;
+
+    if (!claim) {
```

Comment:
When would there be no claim? Should this be a loading spinner or something instead of the "No information to display" message?

Reply:
The about tab isn't displayed at first, so it doesn't actually show until the user switches to the view. So it may end up not being noticeable. A loading indicator is displayed on the Content tab.